src/lib/ngsi10/SubscribeContextResponse.cpp

```diff
@@ -60,6 +60,30 @@ SubscribeContextResponse::SubscribeContextResponse(StatusCode& errorCode)
 
 /* ****************************************************************************
 *
+* SubscribeContextResponse::toJson -
+*/
+std::string SubscribeContextResponse::toJson(void)
+{
+  std::string out     = "";
```

Comment:
Style: excesive whitepace here?

Reply:
[ Depends on what comes after. The '=' may be aligned with the '=' on the following line. If not, then yes ]

Reply:
Correct. In this case neither L66 (above) or L68 (below) has assignation stamente, thus my comment about excesive whitepace.

Reply:
Fixed in 7480bca